erpnext/selling/doctype/customer/customer.py

```diff
@@ -53,6 +54,16 @@ def validate(self):
 		self.flags.old_lead = self.lead_name
 		validate_party_accounts(self)
 		self.validate_credit_limit_on_change()
+		self.check_cgroup_or_terr_change()
+
+	def check_cgroup_or_terr_change(self):
```

Comment:
never use abbreviations like this!!

https://github.com/frappe/erpnext/wiki/Naming-Guidelines